### PR TITLE
121 cs sexy

### PR DESCRIPTION
**Pull request by mmoollllee**
#121 see comment there

### CODE REVIEW

**Comment by GigaHierz** [comment]
Register card
 - did we not say to make the register card smaller? i cant see the login at first

Home Page
- why does it not jump to one but makes the cards smaller?
- when i come back here from the project page suddenly only project A is displayed as you project and B and C as explore projects and mariana is the current user... i don't know whoes responsibility this is... and when i reload, all projects are again under explore projects and username is user

Project Page
- the check boxx for task done is after ckliking two times irritating and moves around when hovering over it and has a weird border

File: src/app/register-card/register-card.component.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { Component, OnInit, NgModule } from "@angular/core";
+import { Component, OnInit, NgModule, Inject, HostBinding, ViewEncapsulation } from "@angular/core";
 import {
   FormControl,
   ReactiveFormsModule,
   Validators,
   FormGroup
 } from "@angular/forms";
 import { ActivatedRoute, Router } from "@angular/router";
+import { Route } from '@angular/compiler/src/core';
```
Comment (GigaHierz) [inline src/app/register-card/register-card.component.ts]: Wird gar nciht verwendet. Auto import ausversehen?

**Comment by mmoollllee** [comment]
>     * did we not say to make the register card smaller? i cant see the login at first
Done!

>     * why does it not jump to one but makes the cards smaller?
Whoot?

>     * when i come back here from the project page suddenly only project A is displayed as you project and B and C as explore projects and mariana is the current user... i don't know whoes responsibility this is... and when i reload, all projects are again under explore projects and username is user
That's due to currentUser State. It's mockdata. But that's fine.

>     * the check boxx for task done is after ckliking two times irritating and moves around when hovering over it and has a weird border
Fixed.

.top-right-icon is positioned right now too.

File: src/app/register-card/register-card.component.ts (hunk 1 above)
Comment (mmoollllee) [reply src/app/register-card/register-card.component.ts]: > Wird gar nciht verwendet. Auto import ausversehen?

Check

**Pull request merged**